src/Scene/NodeProcess.js

```diff
@@ -191,6 +192,14 @@ NodeProcess.prototype.refineNodeLayers = function refineNodeLayers(node, camera,
     }
 };
 
+NodeProcess.prototype.getContentLayers = function getContentLayers(node, camera, params) {
+    if ((node.content.size < params.layersConfig.getGeometryLayers().length) && node.pendingLayers[2] === undefined) {
```

Comment:
This is a quick fix, 
It might be `size === geometryLayers.lenght`, 
When geometry layer returns null the content is filled by nullMesh